### ISSUE TITLE
FocusTrap with ckeditor dialog are not accessible

### ISSUE THREAD

**Issue by TeroOtavia** [otavanopisto/muikku#7003]
FocusTrap with ckeditor dialog are not accessible. Disable FocusTrap completely for now before better solution is found.
